front/src/applications/editor/tools/rangeEdition/components/RangeEditionLeftPanel.tsx

```diff
@@ -107,14 +107,18 @@ const RangeEditionLeftPanel = () => {
     { skip: !infraID }
   );
 
-  const { data: speedLimitTags } =
+  const { data: speedLimitTagsByInfraId } =
     osrdEditoastApi.endpoints.getInfraByInfraIdSpeedLimitTags.useQuery(
       {
         infraId: infraID as number,
       },
       { skip: !infraID }
     );
 
+  const { data: speedLimitTags } = osrdEditoastApi.endpoints.getSpeedLimitTags.useQuery();
+
+  const allSpeedLimitTags = uniq(concat(speedLimitTags, speedLimitTagsByInfraId));
+
```

Comment:
Same here you could use the custom hook.

Reply:
Same case as for MapSettingsSpeedLimits